### PR TITLE
Change main navigation and footer links

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/841

Notes:
- Only displays keplr wallet connect features on desktop, adding them on mobile should be tackled as part of https://github.com/regen-network/regen-web/issues/786
- Does not implement "disconnect" from Keplr as it would have required a lot more work than originally estimated, since in order to achieve this, we might need to use the [AccountStore](https://github1s.com/chainapsis/keplr-wallet/blob/master/packages/stores/src/account/store.ts#L70) from '@keplr-wallet/stores' and change significantly how connecting to keplr is handled in the current code (also see how it's being done on osmosis front-end: https://github.com/osmosis-labs/osmosis-frontend/blob/9c5f8cf5de035348e0aeb38468f1b8a208a9b99d/src/dialogs/connect-wallet.tsx#L255). Similarly, it tries to auto-connect on page load (per the existing implemented behavior) whereas the desired behavior is for the user to connect on demand (and then auto-connect once he/she has connected once). This should be prioritized and addressed as part of a separate issue (@blushi TODO: create issue after discussing this).

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/organisms/RegistryNav.tsx [hunk 1]
```diff
@@ -96,6 +87,12 @@ const RegistryNav: React.FC = () => {
   };
 
   const menuItems: HeaderMenuItem[] = [
+    // TODO: Hide before merging
+    // Add it back once the rNCT basket is created on mainnet
```
[the remark this answers is not in the document]
Comment (mhagel) [reply web-registry/src/components/organisms/RegistryNav.tsx:91]: Yes, that's what I would do. It's nice to have ledger features visible for development and test environments.

File: web-registry/src/lib/wallet.tsx [hunk 2]
```diff
@@ -50,13 +52,30 @@ const defaultClientOptions = {
 
 export const WalletProvider: React.FC = ({ children }) => {
   const [sender, setSender] = useState<Sender>(emptySender);
+  // Because initiating the wallet is asyncronous, when users enter the app, the wallet is seen as not loaded.
+  // This is being used so that we display the "connect wallet" or the connected wallet address
+  // only once we know what's the actual wallet connection status.
+  const [loaded, setLoaded] = useState<boolean>(false);
```
Comment (mhagel) [inline web-registry/src/lib/wallet.tsx:58]: 👍

File: web-registry/src/lib/wallet.tsx [hunk 3]
```diff
@@ -50,13 +52,30 @@ const defaultClientOptions = {
...
   const [txResult, setTxResult] = useState<BroadcastTxResponse | undefined>(
     undefined,
   );
 
   window.onload = async () => {
     if (!sender.address) {
-      await getWallet();
+      try {
+        // This should be updated to use the AccountStore from '@keplr-wallet/stores'
```
Comment (mhagel) [inline web-registry/src/lib/wallet.tsx:66]: Cool.

**Review by mhagel — APPROVED** [review]
Great job.

**Review by victordelval — APPROVED** [review]

**Comment by blushi** [comment]
@Mergifyio backport hambach

**Pull request merged**